k9mail/src/main/java/com/fsck/k9/helper/AppUpdater.kt

```diff
@@ -84,25 +84,11 @@ constructor(
         if (needsUpdateFor(v3_1_7)) {
             v317Update(editor)
         }
-        if (needsUpdateFor(v3_1_11)) {
-            v3111Update(storage, editor)
-        }
         if (needsUpdateFor(v3_1_14)) {
             v3114Update(editor)
         }
     }
 
-    private fun v3111Update(storage: Storage, editor: StorageEditor) {
```

Comment:
why was this removed? Because now we don't handle the passphrase for new keys anymore as in the first proposal? Like No need for the flag?

Reply:
We use it but several updates on same storage key was being buggy. Also the outcome will be the same as one update overwrites the other.